electrical/motor_controller.py

```diff
@@ -144,20 +140,3 @@ def motors(self, omega, vel):
             self.p2.ChangeDutyCycle(dc2)
         else: 
             print("dc1: ", dc1, "and dc2: ", dc2)
-
-    def setup(self):
-        if self.is_sim: 
-            self.motors(0, 0)
-        else:
-            GPIO.setmode(GPIO.BCM)
-            GPIO.setup([self.in1, self.in2, self.in3, self.in4], GPIO.OUT, initial=GPIO.LOW)  
-            GPIO.setup([self.enA, self.enB], GPIO.OUT)  # EnA, EnB
-
-            self.p1 = GPIO.PWM(self.enA, 50)
-            self.p2 = GPIO.PWM(self.enB, 50)
-
-            # Initialize PWM duty cycles as 0
-            self.p1.start(0)
-            self.p2.start(0)
-
-            self.motors(0, 0)
```

Comment:
Currently `robot.py` and `robot_test.py` use this function. Is there an alternative function we can call instead of this?

Reply:
update nvm -- julie pointed out the new setup func in the other pr!